### PR TITLE
Wrong indictor position when set placeholder

### PR DESCRIPTION
**Pull request by cooler333**
Fixed bug [#98](https://github.com/raulriera/TextFieldEffects/issues/98)

### CODE REVIEW

File: TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift [hunk 1]
```diff
@@ -22,8 +22,10 @@ class ExampleTableViewController : UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        _ = textFields.forEach { $0.textAlignment = .center }
```
Comment (raulriera) [inline TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift]: Can you remove this please?

File: TextFieldEffects/TextFieldEffects/YoshikoTextField.swift [hunk 2]
```diff
@@ -87,8 +87,10 @@ import UIKit
         }
     }
 
+    private var _placeholder: String?
```
Comment (raulriera) [inline TextFieldEffects/TextFieldEffects/YoshikoTextField.swift]: Can you move this to the base class please?

File: TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift (hunk 1 above)
Comment (cooler333) [inline TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift]: how to test?

Comment (raulriera) [inline TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift]: You can use a boolean like just a few lines below. Essentially you are changing the example project to make them all center

**Pull request closed without merging**